versions/1/data_stream/manifest.spec.yml

```diff
@@ -48,10 +48,13 @@ spec:
             description: Data type of variable.
             type: string
             enum:
-              - integer
               - bool
+              - email
+              - integer
               - password
               - text
+              - timezone
```

Comment:
time_zone or timezone?

Reply:
Not a strong opinion, what do you prefer? :)

Reply:
I checked the Elasticsearch docs and there are references to time_zone: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-range-query.html

On the other hand I'm not a native speaker :)

Reply:
Let's go with `time_zone` then :)